src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,96 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+import { getNumbers, getItemsToShowIndex } from '../../utils';
+
+type PaginationProps = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (value: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}: PaginationProps) => {
+  const numberOfPages = Math.ceil(total / perPage);
+
+  const [from, to] = getItemsToShowIndex(perPage, currentPage, total);
```

Comment:
Use a more descriptive variable name instead of 'items'. For example, 'itemsToShow'.